__tests__/app.test.js

```diff
@@ -13,30 +13,66 @@ afterAll(() => {
 })
 
 describe('/api/topics', () => {
-    test('200: GET /api/topics - responds with an array of topic objects', () => {
-        return request(app)
-        .get('/api/topics')
-        .expect(200)
-        .then(({ body }) => {
-            const { topics } = body;
-            
-            expect(topics.length).toBe(3);
-            
-            topics.forEach((topic) => {
-                expect(typeof topic.description).toBe('string');
-                expect(typeof topic.slug).toBe('string');
+    describe('STATUS 200', () => {
+        test('200: GET /api/topics - responds with an array of topic objects', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({ body }) => {
+                const { topics } = body;
+
+                expect(topics.length).toBe(3);
+
+                topics.forEach((topic) => {
+                    expect(typeof topic.description).toBe('string');
+                    expect(typeof topic.slug).toBe('string');
+                })
+            })
+        })
+    })
+
+    describe('ERRORS', () => {
+        test('404 returns \'path not found\' for route that does not exist', () => {
+            return request(app)
+            .get('/api/h')
+            .expect(404)
+            .then(({body}) => {
+                expect(body.message).toBe('404: Not found');
             })
         })
     })
 })
 
-describe('', () => {
-    test('404 returns \'path not found\' for route that does not exist', () => {
-        return request(app)
-        .get('/api/h')
-        .expect(404)
-        .then(({body}) => {
-            expect(body.message).toEqual('Path not found');
+describe('/api/articles/:id', () => {
+    describe('STATUS 200', () => {
+        test('200: GET /api/articles/:id - responds with article whose id matches the given id', () => {
+            return request(app)
+            .get('/api/articles/4')
+            .expect(200)
+            .then(({ body }) => {
+                const { article } = body;
+
+                expect(typeof article.author).toBe('string');
```

Comment:
a better jest matcher for testing objects, especially as you'll start testing bigger ones in the next few tickets, is .toMatchObject or .objectContaining. I'd suggest having a go at using them on future tickets